clang/lib/Headers/cpuid.h

```diff
@@ -211,7 +214,11 @@
 /* Features in %edx for leaf 7 sub-leaf 1 */
 #define bit_AVXVNNIINT8   0x00000010
 #define bit_AVXNECONVERT  0x00000020
+#define bit_AMXCOMPLEX    0x00000100
+#define bit_AVXVNNIINT16  0x00000400
 #define bit_PREFETCHI     0x00004000
+#define bit_USERMSR       0x00008000
+#define bit_AVX10_256     0x00080000
```

Comment:
This is different with GCC.

Reply:
Thanks for pointing this out. I was to but forgot. This is bit_AVX10 in gcc. I found in llvm/lib/TargetParser/Host.cpp it sets so.

Reply:
They are identical to check this bit or another in Host.cpp. They are different to user who check the bit through cpuid, because they have different sub leaf and different value.

Reply:
I see. Aligned with gcc: [fd0c860](https://github.com/llvm/llvm-project/pull/80815/commits/fd0c860aab3c78505222c281419a366a36104e37)